src/zenml/integrations/kubeflow/flavors/kubeflow_orchestrator_flavor.py

```diff
@@ -51,11 +51,9 @@ class KubeflowOrchestratorConfig(BaseOrchestratorConfig):
             API. If not set, the hostname will be derived from the Kubernetes
             API proxy.
         kubeflow_namespace: The Kubernetes namespace in which Kubeflow
-            Pipelines is deployed.
+            Pipelines is deployed. Defaults to `kubeflow`.
         kubernetes_context: Optional name of a kubernetes context to run
-            pipelines in. If not set, the current active context will be used.
-            You can find the active context by running `kubectl config
-            current-context`.
+            pipelines in. If not set, will try to spin up a K3d cluster.
```

Comment:
```suggestion
            pipelines in. If not set, will try to spin up a local K3d cluster.
```
Just a little more clarity in case someone isn't familiar with k3d